docs/css/styles.css

```diff
@@ -521,11 +523,32 @@ button:-moz-focusring,
       grid-column-end: 4;
       justify-self: end; } }
 
+.ov-masthead--community {
+  background: url("/images/oVirt_Community_Header.png") left top no-repeat var(--ov--color-brand--100); }
+  .ov-masthead--community .ov-heading--lg {
+    max-width: 400px;
+    margin-left: auto;
+    margin-right: auto; }
+
+.ov-layout-styles {
+  margin: var(--ov--spacer--sm);
+  padding: var(--ov--spacer--sm);
+  max-width: 1080px; }
+  @media (min-width: 992px) {
+    .ov-layout-styles {
+      padding: var(--ov--spacer--md); } }
+
+.ov-content-styles p {
```

Comment:
nitpic but you can actually nest this tags inside the `.ov-content-styles` so you don't have to keep writing the class. The SCSS will output it like you've written it here